### DISCUSSION TITLE
Reply to a Replied Post

### DISCUSSION THREAD

**Discussion by srkgupta** [mattermost/community#88] [Practical Ideas]
An option to reply to a replied post can be provided which when clicked will copy and pre-populate the first line of the conversation with blockquote.  

For example: 

<img width="483" alt="Screenshot 2021-06-24 at 7 19 48 PM" src="https://user-images.githubusercontent.com/1429138/123274489-2f5a0d80-d521-11eb-8b1a-1156a29d51af.png">

If `sysadmin` replies to the replied post from `aaron.peterson`, it will be shown like this:

<img width="483" alt="Screenshot 2021-06-24 at 7 19 40 PM" src="https://user-images.githubusercontent.com/1429138/123274626-4ef13600-d521-11eb-8464-017bb711dd0c.png">

Notice that it does 2 things here:
- Copies and pre-populates the first line of the conversation 
- Tags the user (this is optional)

**Comment by esethna** [comment]
I think we could do quoting better since we can't do threads within threads. There's the option of the github style quote, but we could even do a pop-up menu once you highlight a selection within a message

**Comment by Willyfrog** [comment]
We should have a shortcut (like the one for marking as unread) to quickly reply-quote so a user can go through the conversation clicking on what they want to reply to and at the end just edit the draft with the replies
